src/components/Signal/Signal.tsx

```diff
@@ -85,21 +44,89 @@ const Signal: React.FC<SignalProps> = ({
       {...props}
     >
       <title>{title}</title>
-      <circle cx="8" cy="8" fill={color} r="8" />
-      <g>
-        {paths.map((path, key) => (
-          <path
-            // eslint-disable-next-line react/no-array-index-key
-            key={`${kind}-icon-${key}`}
-            d={path}
-            fill={colors.neutral[0]}
-          />
-        ))}
+      <g style={{ mixBlendMode: 'multiply' }}>
+        <path
+          d="M3.5 8C4.3125 8 5 8.6875 5 9.5V14.5C5 15.3438 4.3125 16 3.5 16C2.65625 16 2 15.3438 2 14.5V9.5C2 8.6875 2.65625 8 3.5 8Z"
+          fill={barColors[0]}
+        />
+      </g>
+      <g style={{ mixBlendMode: 'multiply' }}>
+        <path
+          d="M7 5.5C7 4.6875 7.65625 4 8.5 4C9.3125 4 10 4.6875 10 5.5V14.5C10 15.3438 9.3125 16 8.5 16C7.65625 16 7 15.3438 7 14.5V5.5Z"
+          fill={barColors[1]}
+        />
+      </g>
+      <g style={{ mixBlendMode: 'multiply' }}>
+        <path
+          d="M13.5 0C14.3125 0 15 0.6875 15 1.5V14.5C15 15.3438 14.3125 16 13.5 16C12.6562 16 12 15.3438 12 14.5V1.5C12 0.6875 12.6562 0 13.5 0Z"
+          fill={barColors[2]}
+        />
       </g>
     </svg>
   );
 };
 
+const components = {
+  [SignalKinds.low]: (props) => (
+    <SeverityIcon
+      colors={[colors.severity.low, INACTIVE_BAR_COLOR, INACTIVE_BAR_COLOR]}
+      {...props}
+    />
+  ),
+  [SignalKinds.medium]: (props) => (
+    <SeverityIcon
+      colors={[
+        colors.severity.medium,
+        colors.severity.medium,
+        INACTIVE_BAR_COLOR,
+      ]}
+      {...props}
+    />
+  ),
+  [SignalKinds.high]: (props) => (
+    <SeverityIcon
+      colors={[
+        colors.severity.high,
+        colors.severity.high,
+        colors.severity.high,
+      ]}
+      {...props}
+    />
+  ),
+  [SignalKinds.critical]: (props) => <SeverityIcon {...props} />,
+  [SignalKinds.positive]: ({ size, ...props }: BaseSvgProps) => (
+    <Icon
+      color={ColorTypes.severityPositive}
+      data-testid="ds-severity-icon"
+      name="check-circle-solid"
+      style={{ fontSize: size }}
```

Comment:
```suggestion
      aria-hidden={false}
      style={{ fontSize: size, verticalAlign: 0 }}
```
We want to add `aria-hidden` prop as we don't want those severity icons to be hidden from screen readers.
The `verticalAlign: 0` reset styling from FA and fixes the misalignment between old and new icons